src/views/Proposal.vue

```diff
@@ -185,24 +185,39 @@ watch(
               <span class="text-skin-text text-[16px]" v-text="_rt(proposal.created)" />
             </div>
           </router-link>
-          <div class="flex items-center gap-4 mr-3">
-            <UiTooltip v-if="editable" title="Edit proposal">
-              <a @click="handleEditClick">
-                <IH-pencil />
-              </a>
-            </UiTooltip>
-            <UiTooltip v-if="cancellable" title="Cancel proposal">
-              <UiLoading v-if="cancelling" />
-              <a v-else @click="handleCancelClick">
-                <IS-x-mark />
-              </a>
-            </UiTooltip>
-            <UiTooltip title="View proposal metadata">
-              <a v-if="proposalMetadataUrl" :href="proposalMetadataUrl" target="_blank">
-                <IH-dots-vertical />
+          <UiDropdown class="mr-3">
+            <UiDropdownItem v-if="editable" v-slot="{ active }">
+              <button
+                class="flex items-center gap-2"
+                :class="{ 'opacity-80': active }"
```

Comment:
Can we set this style within the Component itself so we don't need to replicate it elsewhere and we got same style across each dropdown?

Reply:
When "active" is being used? Isn't this similar than "hover"? We can keep it like this if it's to complicated.

Reply:
Headless UI supports accessibility so active will be triggered if you focus in with keyboard, if you open popup and start typing part of word from item etc.